docs/reference/config-options.md

```diff
@@ -39,6 +39,7 @@ settings:
     legalhold: disabled-by-default
     teamSearchVisibility: disabled-by-default
     setEmailVisibility: visible_to_self
+    classifiedDomains: disabled-by-default
```

Comment:
`disabled-by-default` is not a valid config value for `classifiedDomains`

Reply:
Right. I see that when other features were documented in this file, that they referred to a specific commit in the `wire-server` repository so that a pinned version can be inspected for how it was implemented. Should I do the same here? If so, I suppose I should leave out changing this documentation file in the PR and use a follow-up PR that can reference a to-be merged commit of this PR.

Reply:
@smatting , just a gentle reminder of this question.